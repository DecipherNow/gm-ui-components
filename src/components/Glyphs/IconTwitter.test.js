import React from "react";
import { shallow } from "enzyme";

import IconTwitter from "./IconTwitter.svg";

describe("IconTwitter", () => {
  it("matches snapshot", () => {
    const aIconTwitter = shallow(<IconTwitter />);
    expect(aIconTwitter).toMatchSnapshot();
  });
});