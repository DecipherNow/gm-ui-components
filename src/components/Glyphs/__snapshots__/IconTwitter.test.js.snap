// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconTwitter matches snapshot 1`] = `<IconMock />`;
